services/ui-src/src/components/modals/AddEditReportModal.tsx

```diff
@@ -62,6 +63,8 @@ export const AddEditReportModal = ({
       metadata: {
         submissionName,
         lastAlteredBy: full_name,
+        locked: false,
+        previousRevisions: [],
```

Comment:
what are we tracking in `previous revisions`?

Reply:
Previous revisions will be helpful when we go to implement resubmissions. Its not actually used here, but I figured since I'm right here and grabbing it I'll make it easier for everyone who's about to work on it. Correct me if I'm wrong @karla-vm, (I think it was you who implemented this, Sorry if not!) but Previous revisions is an array of the past ids of field data that you've submitted. So after an admin comes back and asks a user to fill out the form again, that gets updated a link to the past answers.